.github/workflows/_Test-OCI-Factory.yaml

```diff
@@ -15,7 +15,58 @@ on:
       - "!src/workflow-engine/**"
       - "!src/cli-client/**"
 
+env:
+  # local path to clone the oci-factory to
+  OCI_FACTORY_DIR: oci-factory/
```

Comment:
```suggestion
```

it's unnecessary - the checkout action will provide the repo at `PWD`

Reply:
I started doing this by convention because later in the build workflow we clone 2 repositories, the rock-repo for the project, and our own repo for the scripts. It is little awkward, but it prevents collisions if the repos had similar directory structures.

Reply:
that won't happen cause those are different jobs. Each job has it own workspace, so there won't be any collisions

Reply:
Here is a sample from the refactored build-rock workflow where it could happen in the lpci job. If we get rid of this type of build, we could also mitigate the collision issue entirely. What do you think?


```
      # Job Setup
      - uses: actions/checkout@v4
        with:
          repository: ${{ env.OCI_FACTORY_REPO }}
          path: ${{ env.OCI_FACTORY_DIR }}
          ref: ${{ env.OCI_FACTORY_BRANCH }}
          fetch-depth: 1

      - uses: actions/setup-python@v5
        with:
          python-version: "3.x"
      - run: pip install -r ${{ env.OCI_FACTORY_DIR }}/src/build-rock/lpci_build/requirements.txt
      

      - name: Clone Repo
        uses: actions/checkout@v4
        with:
          repository: ${{ inputs.rock-repo }}
          path: ${{ env.ROCK_REPO_DIR }}
          ref: ${{ inputs.rock-repo-commit }}
          submodules: 'recursive'

      - name: Build Target
      # TODO: Replace this retry action with bash equivalent for better testing
        uses: nick-fields/retry@v3.0.0

...
``

Reply:
But that's a different job. In that case, it makes sense. But in this job it is unnecessary.

Reply:
Sounds good, I will apply the changes.